x-pack/plugins/actions/server/builtin_action_types/pagerduty.ts

```diff
@@ -64,6 +64,24 @@ const ParamsSchema = schema.object(
     component: schema.maybe(schema.string()),
     group: schema.maybe(schema.string()),
     class: schema.maybe(schema.string()),
+    links: schema.maybe(
+      schema.arrayOf(
+        schema.object({
+          href: schema.uri(),
+          text: schema.maybe(schema.string()),
+        })
+      )
+    ),
+    images: schema.maybe(
+      schema.arrayOf(
+        schema.object({
+          src: schema.uri(),
+          href: schema.maybe(schema.uri()),
+          alt: schema.maybe(schema.string()),
+        })
+      )
+    ),
+    customDetails: schema.maybe(schema.recordOf(schema.string(), schema.string())),
```

Comment:
This will need to be a `schema.nullable()` vs `schema.maybe()`, to prevent "partial updates" of this field when it's updated in ES.  Without it, it's impossible to delete fields in the record.

Reply:
same is true for `links` and `images` as well, ^^^